org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/WorkspaceSymbolHandler.java

```diff
@@ -11,11 +11,13 @@
 package org.eclipse.jdt.ls.core.internal.handlers;
 
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
 
+import org.apache.maven.shared.utils.StringUtils;

```

Comment:
please don't use Maven APIs in non Maven stuff.

Reply:
use org.apache.commons.lang3.StringUtils

Reply:
nice catch!